.circleci/config.yml

```diff
@@ -893,20 +907,26 @@ workflows:
               only: /.*/
 
 
-  # Daily rebuild of our sanitized mysql data image, runs at 12:30 PM EST.
+  # Daily rebuild of our sanitized and super-sanitized mysql data images.
```

Comment:
At first blush, I dont really care about what time populate our DBs. Are you saying that backups happen before 12:30 so we should populate then? Please clarify.

Reply:
The Acquia API becomes available to us at 12:30 pm ET that why the sanitized DB runs in the afternoon. If we change this to 8:48 am ET it will grab a two day old API compared to a day old database.

Reply:
Seems to me like the databases can be 0-24 hours old since we run once per day. Not seeing how it could be older than that. I dont care much where it sits in the range of 0-24.